pages/dkp/kommander/2.3/clusters/custom-domain-certificate/managed/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: layout.pug
+navigationTitle: Managed Cluster 
+title: Customize a domain and certificate in a Managed Cluster
+menuWeight: 20
+excerpt: Configure a custom domain and certificate in a Managed or Attached Cluster
+beta: false
+enterprise: false
+---
+
+This section describes how to enable custom domains and certificates in your **managed and attached** clusters. Refer to the [Customize a domain and certificate in the Management Cluster][management] documentation, if you want to set up custom domains and certificates for your **Management Cluster** during the installation of DKP.
+
+To customize the domain or certificate for a specific cluster after the installation of DKP, adapt or create an API `yaml` that will allow Kommander to implement the established adjustments on top of the default or any previous configuration you have made.
+
+## Configure custom domains or custom certificates
+
+<p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster of the environment where you want to customize the domain or certificate by setting the <code>KUBECONFIG=<path></code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
+
+Use the API yaml to customize the domain (via the `hostname` field), the certificate (via the `issuerRef` or `certificateSecretRef` field), or both. For this, refer to the following examples:
+
+1.  Create or update and apply the `KommanderCluster` object with the wanted ingress. Remember to specify the cluster in the `kubeconfigRef` name field.
```

Comment:
> Remember to specify the cluster in the `kubeconfigRef` name field.

This might be a bit misleading, because it's not related to domains or certificates. The property is set when creating (=attaching) the cluster (or during installation for the Management Cluster)

Reply:
I agree. Maybe we could try to provide `patch` examples that would only modify the parts that are changing the ingress configuration.